spec/integration/worksheet.spec.js

```diff
@@ -659,6 +659,48 @@ describe('Worksheet', () => {
       });
     });
 
+    context('when the worksheet name contains illegal characters', () => {
+      it('throws an error', () => {
+        const workbook = new ExcelJS.Workbook();
+
+        const invalidCharacters = ['*', '?', ':', '/', '\\', '[', ']'];
+        const expectedErrors = invalidCharacters.map(
```

Comment:
I think the array creation and the for-index-loop distract a little bit from what is being tested, I think we should simplify to this:
```js
for (const invalidCharacter of invalidCharacters) {
  expect(() => workbook.addWorksheet(invalidCharacter)).to.throw(
    `Worksheet name ${invalidCharacter} cannot include any of the following characters: * ? : \\ / [ ]`
  );
}
```